README.md

```diff
@@ -126,3 +174,22 @@ Standard](https://www.researchgate.net/publication/228971719_A_syntactic_specifi
 by Flor Narciso et al.  Many aspects of the grammar have been added to,
 modified, and in cases entirely rewritten to better support lark grammars and
 transformers.
+
```

Comment:
```suggestion

Special thanks to the blark contributors:
* @engineerjoe440 

```

Reply:
Hah! I almost feel guilty committing this suggestion... Thank you for the shout-out, but I've done so little for this wonderful project. :blush: